README.md

```diff
@@ -215,6 +215,45 @@ pipeline {
 }
 ```
 
+#### Delete build artifacts matching patterns
+The following configuration has a single rule that utilizes the `Match every build` condition and deletes specific build artifacts based on Ant-style glob patterns defined in the `Include Patterns/Exclude Patterns`.
+
+Here's an example using the include pattern `"**"`, exclude pattern `"**/*.log"` to delete all files except log files:
+
+```groovy
+properties(
+    [
+        buildDiscarder(BuildHistoryManager(
+                [
+                    [
+                        actions: [DeleteArtifactsMatchingPatterns(excludePatterns: '**/*.log', includePatterns: '**')],
+                        conditions: [MatchEveryBuild()]
+                    ]
+                ]
+            )
+        )
+    ]
+)
+
+node {
+
+        sh '''
+        rm -rf *
+        touch test.xml
+        touch testLog.log
+        touch testTxt.txt
+        mkdir -p testFolder
+        touch testFolder/test1.xml
+        touch testFolder/testLog1.log
+        touch testFolder/testTxt1.txt
+        '''
+        archiveArtifacts artifacts: '**', followSymlinks: false
```

Comment:
Here is the changes in wiki, because Github does not allow to send pull request to the wiki. https://github.com/nurgul212/build-history-manager-plugin-wiki/compare/artifacts-pattern?expand=1

In wiki, `Delete-artifacts-matching-patterns-action.md` , there is an example to present build history before and after running the configuration with this part.